packages/events/src/service/indexing/indexing.ts

```diff
@@ -176,18 +176,28 @@ export async function deleteEventIndex(event: EventDocument) {
 }
 
 export async function getIndexedEvents() {
+  // @ToDo: remove hardcoded events
+  const events = ['BIRTH', 'TENNIS_CLUB_MEMBERSHIP']
+  const indexedEventsPromises = events.map(async (event) =>
+    getIndexedEventsByIndex(getEventIndexName(event))
+  )
+  const indexedEventsArrays = await Promise.all(indexedEventsPromises)
+  return indexedEventsArrays.flat()
+}
+
+export async function getIndexedEventsByIndex(index: string) {
   const esClient = getOrCreateClient()
 
   const hasEventsIndex = await esClient.indices.exists({
-    index: getEventAliasName()
+    index
```

Comment:
The events alias has all underlying indices connected to it, so it should contain all events